src/main/kotlin/me/nalab/api/core/TimeUtil.kt

```diff
@@ -0,0 +1,34 @@
+package me.nalab.api.core
+
+import java.time.Clock
+import java.time.Instant
+import java.time.ZoneId
+import java.time.format.DateTimeFormatter
+
+object TimeUtil {
+
+    private var clock: Clock? = null
+
+    fun toInstant(): Instant {
+        var current = Instant.now()
+        if (clock != null) {
+            current = Instant.now(clock)
+        }
```

Comment:
val로 쓰고 if statement 로 처리하기
```suggestion
        val current = if (clock == null) Instant.now() else Instant.now(clock)
```

Reply:
[6bfa224](https://github.com/depromeet/na-lab-server/pull/395/commits/6bfa2249fd1e70d57d229621a0c44698cc14695c)